operator/builtin/input/journald/journald.go

```diff
@@ -123,6 +129,38 @@ func (operator *JournaldInput) Start() error {
 		return err
 	}
 
+	operator.startPoller(ctx)
+	return nil
+}
+
+// startPoller kicks off a goroutine that will poll journald periodically,
+// checking if there are new files or new logs in the watched files
+func (operator *JournaldInput) startPoller(ctx context.Context) {
+	go func() {
```

Comment:
Even though this function is mostly just running `poll`, which manages its own waiting, I think we should increment the wait group here, and immediately defer `wg.Done()`, so that we are not at risk of leaving the ticker running.

Reply:
resolved here https://github.com/observIQ/stanza/pull/380/commits/9ce72a10eb865a27c2a1a573eb8596cf1edd56ea